socorro/unittest/external/es/test_crashstorage.py

```diff
@@ -598,12 +599,14 @@ def mock_index(*args, **kwargs):
             if "bogus-field" in kwargs["body"]["processed_crash"]:
                 raise elasticsearch.exceptions.TransportError(
                     400,
-                    "RemoteTransportException[[i-f94dae31][inet[/172.31.1.54:"
-                    "9300]][indices:data/write/index]]; nested: "
-                    "MapperParsingException[failed to parse "
-                    "[processed_crash.bogus-field]]; nested: "
-                    "NumberFormatException[For input string: "
-                    '"18446744073709480735"]; ',
+                    (
+                        "RemoteTransportException[[i-f94dae31][inet[/172.31.1.54:"
+                        "9300]][indices:data/write/index]]; nested: "
+                        "MapperParsingException[failed to parse "
+                        "[processed_crash.bogus-field]]; nested: "
+                        "NumberFormatException[For input string: "
+                        '"18446744073709480735"]; '
+                    ),
```

Comment:
If these tests weren't mocked out, then they would tell us what happens when we update to a recent version of Elasticsearch. We should probably redo these tests so they run against an actual Elasticsearch.